esss_environment.devenv.yml

```
@@ -49,13 +29,13 @@ dependencies:
   {% if TEST_QMXGRAPH %}
   - conda-wrappers
 
-  - cherrypy==7.1.0
-  - phantomjs>=2.1
+  - cherrypy>=18.0.0
   - pytest-mock>=1.4.0
   - pytest-qt>=2.1.0
-  - pytest-selenium>=1.2.1,<2
+  - pytest-selenium>2
   - pytest-timeout>=1.0.0
   - pytest-xdist>=1.15
+  - selenium >=4.0.0, <4.10.0
```

Recent versions o selenium (>=4.10.0) are currently incompatible with pytest-selenium (some constructor changed argument list).

---

Nice explanation.

Let's add this as a comment in the code.

```suggestion
  # Recent selenium versions (>=4.10.0) are currently incompatible with pytest-selenium (some constructor changed argument list).
  - selenium >=4.0.0, <4.10.0
```